src/literature/components/ReferenceItem.jsx

```diff
@@ -5,24 +5,25 @@ import { List } from 'antd';
 import { Link } from 'react-router-dom';
 
 import AuthorList from './AuthorList';
-import DOIList from './DOIList';
-import PublicationInfo from './PublicationInfo';
 import Latex from '../../common/components/Latex';
-import ArxivEprintList from './ArxivEprintList';
 
 class ReferenceItem extends Component {
   static renderTitle(reference) {
     const recordId = reference.get('control_number');
     const title = reference.getIn(['titles', 0, 'title']);
+    let label = reference.get('label');
+
+    label = label ? `[${label}] ` : '';
```

Comment:
you can use `.get('label', '')` instead of `?`. Also I think reassign is not very good thing to do.

Reply:
that's a good one :)